### PR TITLE
Kill customize_export_old.html

### PR DESCRIPTION
**Pull request by orangejenny**
It's difficult to decide when to stop, once you start pulling on the thread of "delete old exports code". This removes BaseExportView, its descendants, and the related html and js.

@esoergel 
code buddies @emord / @millerdev

### CODE REVIEW

**Review by emord — APPROVED** [review]
Looks good, but looks like a big change. Are you planning to go through any QA on staging for thsi?

**Comment by orangejenny** [comment]
@emord I'm not especially worried about it - it's almost entirely deleting views that aren't referenced. I did click through the exports section locally.

Also realizing now that the overall diff for this is misleading due to deleting models and then renaming them. Commit by commit is the way to go.

**Review by millerdev — APPROVED** [review]

File: corehq/apps/export/static/export/js/customize_export.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-hqDefine('export/js/customize_export_new', function() {
```
Comment (esoergel) [inline corehq/apps/export/static/export/js/customize_export.js]: I'm a little hesitant about this change - there's still old export stuff and new export stuff, and I think it's useful to keep it obvious that these files belong to the latter.

Comment (orangejenny) [reply corehq/apps/export/static/export/js/customize_export.js]: Okay, I'm happy to revert this commit.

**Review by esoergel — APPROVED** [review]
thanks!

**Pull request merged**